packages/suite/src/components/suite/Preloader/SuiteLayout/MobileMenu/MobileMenuActions.tsx

```diff
@@ -0,0 +1,109 @@
+import { useMemo } from 'react';
+
+import styled from 'styled-components';
+
+import type { Route } from '@suite-common/suite-types';
+
+import { setDiscreetMode } from 'src/actions/settings/walletSettingsActions';
+import { goto } from 'src/actions/suite/routerActions';
+import { Translation } from 'src/components/suite';
+import { findRouteByName } from 'src/utils/suite/router';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { useCustomBackends } from 'src/hooks/settings/backends';
+import { useGuide } from 'src/hooks/guide/useGuide';
+
+import { MobileActionItem } from './MobileActionItem';
+
+const MobileWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px;
+`;
+
+interface MobileMenuActionsProps {
+    closeMobileNavigation?: () => void;
+}
+
+export const MobileMenuActions = ({ closeMobileNavigation }: MobileMenuActionsProps) => {
+    const activeApp = useSelector(state => state.router.app);
+    const notifications = useSelector(state => state.notifications);
+    const discreetMode = useSelector(state => state.wallet.settings.discreetMode);
+    const allowPrerelease = useSelector(state => state.desktopUpdate.allowPrerelease);
+    const enabledNetworks = useSelector(state => state.wallet.settings.enabledNetworks);
```

Comment:
I know, was one of the people working on that file and I fully agrree! 🙏  Tried to minimize the amount of time spent on the mobile components and refactors in general. It's more comfortable to do it as separate PRs, since sometimes adding a selector means changing like 10 files and this PR is already enormous...

Reply:
Yes, you are right, let's do it later and as separate PR. It's definitely not a priority.